engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/process/FlinkCustomNodeContext.scala

```diff
@@ -7,6 +7,7 @@ import scala.concurrent.duration.FiniteDuration
 import scala.reflect.ClassTag
 
 case class FlinkCustomNodeContext(metaData: MetaData,
+                                  // TODO: it can be used in state recovery - make sure that it won't change during renaming of nodes on gui
                                   nodeId: String,
```

Comment:
I don't know how many raw Strings are in NK, but maybe it would be a good idea to create a value object here?

Reply:
It will be a painfull change. This context is used in all custom transformers in user's implementations. Also I think that it will be a good moment to add value object when we will split nodeId into nodeName and nodeId. For now it is mixed.